File: app/src/main/java/com/pronab/sunbank/model/Transaction.java
package com.pronab.sunbank.model;

/**
 * Created by delaroy on 7/3/17.
 */

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Transaction {
    @SerializedName("id")
    @Expose
    private int id;

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    @SerializedName("description")
    @Expose
    private String description;
    @SerializedName("amount")
    @Expose
    private Float amount;
    @SerializedName("effectiveDate")
    @Expose
    private String effectiveDate;

    public Float getAmount() {
        return amount;
    }

    public void setAmount(Float amount) {
        this.amount = amount;
    }

    public String getEffectiveDate() {
        return effectiveDate;
    }

    public void setEffectiveDate(String effectiveDate) {
        this.effectiveDate = effectiveDate;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }
}

File: app/src/main/java/com/pronab/sunbank/model/Transaction.kt
package com.pronab.sunbank.model

/**
 * Created by delaroy on 7/3/17.
 */

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Transaction {
    @SerializedName("id")
    @Expose
    var id: Int = 0

    @SerializedName("description")
    @Expose
    var description: String? = null
    @SerializedName("amount")
    @Expose
    var amount: Float? = null
    @SerializedName("effectiveDate")
    @Expose
    var effectiveDate: String? = null
}
